src/config/tsconfig.ts

```diff
@@ -15,6 +15,7 @@ const createTSConfig = (): string => {
   for (const filepath of declarationFiles) {
     const moduleName = basename(filepath)
       .replace(/\.d\.ts$/, '')
+      .replace('-', '/')
```

Comment:
This is bad, if the module is called `check-for-winner` and we are typing it we will have an issue.

Make this better